xarrayms/table_cache.py

```diff
@@ -80,8 +72,6 @@ def acquire(self, locktype):
         else:
             raise ValueError("Invalid lock type %d" % locktype)
 
-        self.refcount += 1
-
     def release(self, locktype):
         if locktype == READLOCK:
             self.readlocks -= 1
```

Comment:
Potential race check then update race condition here?

Reply:
No, the refcount isn't used for anything interesting at the moment